frontend/src/container/PipelinePage/PipelineListsView/TableComponents/DragAction.tsx

```diff
@@ -0,0 +1,19 @@
+import { Switch } from 'antd';
+import React from 'react';
+
+import { HolderOutlinedIcon, LastActionColumn } from '../styles';
+
+function DragAction(): React.ReactElement {
+	return (
+		<LastActionColumn>
+			<span>
+				<Switch />
```

Comment:
why this is uncontrolled component ?

Reply:
@palashgdev - Don't have information about what happen on switch.